cli/js/lib.deno.ns.d.ts

```diff
@@ -1229,8 +1229,10 @@ declare namespace Deno {
     /** **UNSTABLE**: Match behavior with Go on Windows for `mode`.
      *
      * The underlying raw `st_mode` bits that contain the standard Unix
-     * permissions for this file/directory. */
+     * permissions for this file/directory (masked to `0o7777`). */
```

Comment:
0o777 ?

Reply:
No, we don't mask out the suid/guid/sticky bits.